### DISCUSSION TITLE
Getting to work with SECTR

### DISCUSSION THREAD

**Discussion by BJPickles** [visualdesigncafe/nature-renderer#15] [Help]
Hey there, my apologies, this is probably something I've done or setup weirdly etc. But basically I've had a weird experience. I setup Nature Renderer as usual following the quick start, everything worked fine aside from a weird artifacting with the post processing etc.

I've since booted up my project a few days later (haven't worked on it since) to find that Nature Renderer isn't showing any of the vegetation anymore which is odd.
This is just a regular Gaia Pro terrain setup with SECTR terrain streaming.

To note a couple of things:
- Camera is instantiated at runetime as it's a multiplayer game (tagged MainCamera).
- Map is empty aside from a few gameobjects and loads "terrain scenes" via SECTR at runtime.
- Each SECTR scene has its own terrain with the "Nature Renderer" script on it.
- The vegetation uses a single material & (copy) of the default "Standard" shader which has been patched.

![image](https://github.com/visualdesigncafe/nature-renderer/assets/48354382/a81bc2b7-b8d4-44d4-bbdd-1eb4bd732d6c)

I have checked the individual SECTR scenes and they render the vegetation in scene, but during runtime - when they've been loaded, they don't show the trees even in scene view.

I also tried disabling / enabling the Nature Renderer script, which reveals the expected vegetation:

https://github.com/visualdesigncafe/nature-renderer/assets/48354382/53a76c91-335a-4f52-a6e0-a62b5416cece

**Comment by BJPickles** [comment]
Just to add to this, the Nature Instances _are_ rendered both in scene & game view

![image](https://github.com/visualdesigncafe/nature-renderer/assets/48354382/16776175-4f10-4f05-9d6c-a90a4b9a4088)

![image](https://github.com/visualdesigncafe/nature-renderer/assets/48354382/13c79b73-e492-403c-b5d8-b2b3e7f6dbf3)

**Comment by maxplooi** [comment]
Thank you for reporting this. It looks like a compatibility issue with SECTR. I don't have the SECTR asset myself so I can't test it directly for you, but I am sure that it is fixable.

First of all, could you download the latest version of Nature Renderer by going to **Help > Nature Renderer > Check for Updates**?

Then, could you open the **Advanced** section of the Nature Renderer component and make sure that all the settings there are enabled?

Lastly, could you open the [Nature Renderer Profiler](https://visualdesigncafe.com/nature-renderer/docs/2022/profiling) to check if the camera and objects are registered there? You can open it by going to **Window > Nature Renderer > Profiler**.

Please let me know the result of the above and I'll do my best to help you out.
